main/luanti/template.py

```diff
@@ -59,30 +60,31 @@
 
 
 def check(self):
-    self.do("bin/minetest", "--run-unittests")
-    self.do("bin/minetestserver", "--run-unittests")
+    self.do("bin/luanti", "--run-unittests")
+    self.do("bin/luantiserver", "--run-unittests")
 
 
 def post_install(self):
     self.install_file(
         "minetest.conf.example",
-        "etc/minetest",
+        "etc/luanti",
         name="minetest.conf",
     )
     # dead symlink
-    self.uninstall("usr/share/minetest/client/shaders/Irrlicht")
+    self.uninstall("usr/share/luanti/client/shaders/Irrlicht")
 
 
-@subpackage("minetest-common")
+@subpackage("luanti-common")
 def _(self):
     self.subdesc = "common files"
+    self.provides = [self.with_pkgver("mintest-common")]
 
-    return ["usr/share/minetest/builtin"]
+    return ["usr/share/luanti/builtin"]
 
 
-@subpackage("minetest-server")
+@subpackage("luanti-server")
 def _(self):
     self.subdesc = "server"
-    self.depends = [self.with_pkgver("minetest-common")]
+    self.depends = [self.with_pkgver("luanti-common")]
```

Comment:
server is missing a provides...

Reply:
the install process adds symlinks mintest->luanti and minetestserver->luantiserver. those get picked up by the pos-install hooks, adding them manually seemed redundant

Reply:
? that doesn't make luanti-server package provide the minetest-server name (just like every other package in this template does with your changes)

Reply:
true